examples/ssh/nginx_stats.js

```diff
@@ -0,0 +1,291 @@
+
+/**
```

Comment:
This driver seems to depend on the server host as well as the nginx configuration so maybe we should remove to avoid confusion?

Reply:
Still unable to test solution due to OSX prompt being different

eg, no 'systemctl', 'lscpu' 
as well as different config file and log files locations 


PS: Use camelCase

Reply:
file removed

Reply:
I added a comment that this driver is applicable for linux based systems